services/blockchain-connector/tests/constants/blocks.js

```diff
@@ -13,80 +13,174 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const block = {
+const blockWithTransaction = {
```

Comment:
Please consider using `Object.freeze()` on all the fixtures.

Reply:
Applied.